lib/EpicPlayer.js

```diff
@@ -10,14 +10,30 @@ Structure.extend(
         this.twentyFourSeven = false;
       }
 
+      /**
+       * Sets the resume message as a property of the player so it can be fetched on pause
+       * @param {Message} message
+       */
+      setResumeMessage(message) {
+        if (this.pausedMessage) this.pausedMessage.delete();
```

Comment:
This has 2 issues. 

> this.pausedMessage
will return true even if the message was already deleted. so probably check using another way.

Secondly, it would be better if the deletion happens before the resume message is sent instead of after. Cuz I tested how it looks and it was looking weird when it has duplicated messages and then the pause get deleted after a second or so. So it would look better if pause is deleted before sending resume embed.

Reply:
> will return true even if the message was already deleted. so probably check using another way.
^^

As discord.js starting version 13.4 had [deprecated the Structure#deleted getter and removed tracking functionality of all deleted stuff](https://github.com/discordjs/discord.js/issues/7091), we will have to track it ourselves if we're gonna insist to reduce invalid API call in case the original message was deleted but it still in cache.